specs/driver_spec.rb

```diff
@@ -76,5 +81,53 @@
       driver = RideShare::Driver.new(id: 54, name: "Rogers Bartell IV", vin: "1C9EVBRM0YBC564DZ")
       driver.average_rating.must_equal 0
     end
-  end
-end
+  end # average_rating
+
+
+  describe 'total_revenue' do
+    it "calculates take home pay for driver" do
+      trips = [
+        RideShare::Trip.new({cost: 10, rating: 3, start_time: Time.parse('2015-05-20T14:14:00+00:00'),
+          end_time: Time.parse('2015-05-20T14:30:00+00:00')}),
+        RideShare::Trip.new({cost: 4, rating: 3, start_time: Time.parse('2015-05-20T14:14:00+00:00'),
+          end_time: Time.parse('2015-05-20T14:30:00+00:00')}),
+        RideShare::Trip.new({cost: 6, rating: 3,start_time: Time.parse('2015-05-20T14:14:00+00:00'),
+          end_time: Time.parse('2015-05-20T14:30:00+00:00')}),
+      ]
+
+      data = {
+        id: 3,
+        vin: 'kkkkkkkkkkkkkkkkk',
+        name: 'Lola',
+        trips: trips
+      }
+
+      driver = RideShare::Driver.new(data)
+      driver.total_revenue.must_equal 12.04
+    end # calculate total rev
+  end # total_revenue
+
+  describe 'average_revenue' do
+    it "can calculate the average pay per hour spent driving" do
+
```

Comment:
Other questions that I would like to see these tests ask, for all the wave 1 methods (on both `Driver` and `Passenger`):

- What happens if the driver or passenger has no trips?
- What happens if the driver or passenger has an incomplete trip?